app/components/moderation-list-row/component.js

```diff
@@ -55,19 +58,33 @@ export default Component.extend({
         return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
     }),
 
-    relevantDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
+    // date of preprint acceptance or rejection
+    acceptedRejectedDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
         return this.get('gtDay') ?
             moment(this.get('submission.dateLastTransitioned')).format('MMMM DD, YYYY') :
             moment(Math.min(Date.parse(this.get('submission.dateLastTransitioned')), Date.now())).fromNow();
     }),
 
-    // translations
-    statusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
+    // translations for moderator action label
+    reviewedOnLabel: computed('submission.reviewsState', 'gtDay', 'noActions', 'acceptedRejectedDate', 'latestActionCreator', function() {
```

Comment:
Thoughts on my comment from the last pass?
>`gtDay` is now (even more) confusingly named. Do you think it would make sense to move it and `acceptedRejectedDate` into this property? It would closer parallel `submittedOnLabel` and reduce the number of interdependent properties.

Reply:
can you clarify your comment, please? What do u mean by closer parallel?

Reply:
I was suggesting getting rid of the `gtDay` and `acceptedRejectedDate` computed properties, and doing those computations inside `reviewedOnLabel` instead.

The main benefit would be reducing the amount of code and making it more readable.

Another benefit would be that `reviewedOnLabel` would have a more similar structure to `submittedOnLabel`. They would be closer to paralleling each other. That's good because they do almost the same thing, and doing similar things in a similar way makes it easier to understand and improve the code.

Do you think that would make sense to do?

Reply:
yes sure.. would look better